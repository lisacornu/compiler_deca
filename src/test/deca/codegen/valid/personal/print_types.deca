// Description:
//    Ecriture des diff√©rents types dans la console
//
// Resultats:
//    String : Bonjour
//    12
//    2.3
//    1 test 3.3
//
// Historique:
//    cree le 06/01/2025

{
    print("String : ");
    println("Bonjour");

    print("Entier : ");
    println(12);

    print("Float : ");
    println(2.3);

    print("Plusieurs : ");
    println(1, " test ", 3.3);
}
