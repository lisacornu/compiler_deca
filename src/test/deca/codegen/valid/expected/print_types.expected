String : Bonjour
12
2.3
1 test 3.3